app/main.py

```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+    for i in phrase:
```

Comment:
i - is not good name
Also: 

> The function is case insensitive

Reply:
name changed to symb
Aded .lower() to checker